### PR TITLE
Fix nlp faculty

### PR DESCRIPTION
**Pull request by bplank**
add more female NLP faculty, fix two affiliations (VU Amsterdam is different from University of Amsterdam), add University of Amsterdam and UPF faculty (working in NLP)

# Contributing to CSrankings

Thanks for contributing to CSrankings! Here are some guidelines to getting your pull request accepted.

**Inclusion criteria**

- [X ] Make sure that any faculty you add meet the inclusion
criteria. Eligible faculty include only full-time, tenure-track
faculty members on a given campus who can advise PhD students in
Computer Science. Faculty not in a CS department or similar who can
advise PhD students in CS can be included regardless of their home
department.

**Updating an affiliation or home page**

- [X ] Update affiliations, home pages, and Google Scholar entries by modifying `csrankings.csv`. For the Google Scholar entry, just use the alphanumeric identifier in the middle of the URL. If none is there, put NOSCHOLARPAGE.

**Adding one or more faculty members (including an entire department)**

- [X ] If the department is not yet listed in CSrankings, the entire faculty needs to be added (not just one faculty member).

added members

- [X ] Enter each faculty member's [DBLP](http://dblp.org) name, home page, and Google Scholar entry (just the alphanumeric identifier, not the whole URL) by modifying `csrankings.csv`; include disambiguation suffixes like 0001 as needed. If the faculty entry is currently ambiguous, please do not include them. Send mail to the DBLP maintainers (dblp@dagstuhl.de) with a few publications by a particular faculty member; also, open an issue so that when the DBLP database is updated, that faculty member's information can be added.

- [ ] If DBLP has multiple entries for this person, all of them need to be listed. If an alias is not already present in `dblp-aliases.csv`, add it.

Unchanged 

- [ ] If the institution you are adding is not in the US,
update `country-info.csv`.

added University of Amsterdam

### CODE REVIEW

File: csrankings.csv [hunk 1]
```diff
@@ -3361,7 +3365,7 @@ Dirk Beyer 0001,LMU Munich,https://www.sosy-lab.org/people/beyer,AFOIl1IAAAAJ
 Dirk Englund,Massachusetts Institute of Technology,http://www.rle.mit.edu/qp,ZFpENKoAAAAJ
 Dirk Fahland,TU Eindhoven,https://www.win.tue.nl/~dfahland,qLkK5JAAAAAJ
 Dirk Grunwald,University of Colorado Boulder,http://systems.cs.colorado.edu/people/faculty/dirk-grunwald,3Ws6G2AAAAAJ
-Dirk Hovy,University of Copenhagen,http://www.dirkhovy.com,7xluaTAAAAAJ
+Dirk Hovy,Bocconi University,http://www.dirkhovy.com,7xluaTAAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:3368]: Bocconi is not in the database, so this entry should be removed.

File: csrankings.csv [hunk 2]
```diff
@@ -4397,6 +4402,7 @@ Geehyuk Lee,KAIST,http://hcil.kaist.ac.kr/?page_id=349,wBXkmcQAAAAJ
 Geert Jan Houben,TU Delft,http://www.wis.ewi.tudelft.nl/houben,7SLMWEcAAAAJ
 Geevarghese Philip,CMI,http://www.cmi.ac.in/~gphilip,3QM3vr4AAAAJ
 Geir E. Dullerud,Univ. of Illinois at Urbana-Champaign,https://mechanical.illinois.edu/directory/faculty/dullerud,afhaKpYAAAAJ
+Gemma Boleda,Pompeu Fabra University,http://gboleda.utcompling.com/,NFJ9kUEAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: AFAICT the entire faculty of Pompeu Fabra have not been included, which is a requirement for adding any new institution.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 1 above)
Comment (bplank) [reply csrankings.csv:3368]: rm-ed

**Comment by bplank** [comment]
Hi @emeryberger 
I unrolled the addition of UPF and removed the other entry.

File: csrankings.csv [hunk 3]
```diff
@@ -1300,6 +1302,7 @@ Augusto Casaca,Universidade de Lisboa,http://comp.ist.utl.pt/ajc,4Fsc1cQAAAAJ
 Augusto Ciuffoletti,University of Pisa,http://www.di.unipi.it/~augusto,H3qOnAgAAAAJ
 Augusto Sampaio,UFPE,http://www.cin.ufpe.br/~acas,03R9LnwAAAAJ
 Aura Conci,UFF,http://www2.ic.uff.br/~aconci,lojRGVgAAAAJ
+Aurélie Herbelot,University of Trento,http://aurelieherbelot.net/,Sby4DYoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv]: Please remove unless entire faculty of Trento are added.

File: country-info.csv [hunk 4]
```diff
@@ -218,3 +219,5 @@ Seoul National University,asia
 Middlesex University,europe
 Newcastle University,australasia
 University of Surrey,europe
+Tilburg University,europe
```
Comment (emeryberger) [inline country-info.csv]: Please remove this and the next entry unless all CS faculty are added.

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 3 above)
Comment (bplank) [reply csrankings.csv]: removed entries. it is not easy to see all

File: country-info.csv (hunk 4 above)
Comment (bplank) [reply country-info.csv]: added all cs faculty.

**Comment by bplank** [comment]
Hi @emeryberger 
I added the faculty from Tilburg and unrolled Tn as that wasn't straightforward to determine..
Barbara

File: csrankings.csv [hunk 5]
```diff
@@ -3638,6 +3642,7 @@ Eisa Zarepour,UNSW,https://research.unsw.edu.au/people/dr-eisa-zarepour,MgSbGaYA
 Eitan Grinspun,Columbia University,http://www.cs.columbia.edu/~eitan,0_vvBZkAAAAJ
 Eitan Yaakobi,Technion,http://www.cs.technion.ac.il/people/yaakobi,Viv2E9AAAAAJ
 Ekaterina Lebedeva,Australian National University,https://cecs.anu.edu.au/people/ekaterina-lebedeva,3Ey9ZWIAAAAJ
+Ekaterina Shutova,University of Amsterdam,https://www.cl.cam.ac.uk/~es407/,jqOFBGoAAAAJ
```
Comment (emeryberger) [inline csrankings.csv:3645]: Is there a web page for this person at Amsterdam?

**Review by emeryberger — CHANGES_REQUESTED** [review]

File: csrankings.csv (hunk 5 above)
Comment (bplank) [reply csrankings.csv:3645]: yes at the institute (albeit her
personal page is still hosted at cam)

I've updated it

**Comment by emeryberger** [comment]
OK, please resolve conflicts (in a minute).

**Comment by bplank** [comment]
I am closing this now, I pulled from the main gh-pages and merged locally and re-open a new pull request to perform the update

**Pull request closed without merging**